.eslintignore

```diff
@@ -3,3 +3,9 @@
 **/staticBundle.js
 **/main.js
 Libraries/vendor/**/*
+RNTester/**/*
```

Comment:
Note this is not ignored upstream - https://github.com/facebook/react-native/blob/master/.eslintignore